### ISSUE TITLE
Focus indicator styles

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1065]
As [similarly discussed for Omeka S](https://github.com/omeka/omeka-s/issues/2316), Seasons, Center Row, Freedom, Lively, and Thanks, Roy override system focus outlines. [Higher-level guidance from WCAG 2.2](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance) dictates that focus indicators, if overriding user agent styles, should have at least 2px perimeters. Currently, these themes utilize thin outlines that fall short of this guidance, and in some cases, have pretty low visibility.

**Comment by kimisgold** [comment]
Berlin, Seasons, Center Row, and Thanks, Roy all have had their custom focus outlines removed or updated to 2px weight and are ready for a look on their default branches. I'll tag @nelsonamaya82 to update Lively and Freedom according to preference.

**Comment by nelsonamaya82** [comment]
Custom focus styles were removed in Freedom and Lively. Now they use default user agent styles.

**Comment by aviannamiller** [comment]
@kimisgold, @nelsonamaya82, thank you for the changes. 

Issues remain for the following:
- Thanks, Roy
  - Subnav toggles on main menu, mobile too
  - Mobile menu toggle for primary nav
  - Search form (text input, advanced options button, submit button)
  - Text inputs elsewhere
- Center Row
  - Search toggle and submit buttons, as well as a few other elements here and there (e.g. featured slide pagination buttons) receive the user agent focus instead
- Berlin
  - Mobile menu: no apparent outlines for parent links. For standalone links or subnav items, an indicator is present, but just as a sort of bottom border on the link container.
- Freedom and Lively
  - Same as Classic, focus indicators become a bit lost for "Previous" links in mobile nav. They seem to almost go behind the background of the links?

**Comment by kimisgold** [comment]
Thanks Roy, Center Row, and Berlin should be ready for a look.

**Comment by aviannamiller** [comment]
Changes look good. For Berlin, the top and left side of the outlines are cut off on resource links under front page featured content and the recently added list:

<img width="467" height="128" alt="Detail of featured item container with link to item, 'Untitled,' below. Item link has focus, but the focus indicator only borders visibly on the bottom and right of the link." src="https://github.com/user-attachments/assets/5e863e20-0584-45d7-9d5c-3274dd35a657" />

I believe you noted these are user agent outlines coming through here now, but if there is a way to adjust this, that would be great.

**Comment by kimisgold** [comment]
There are a lot of instances of `overflow: hidden` that cause this kind of clipping. I've replaced those styles with `display: flow-root`, as it'll allow the container to extend to include floated elements without causing the clipping.

**Comment by aviannamiller** [comment]
That's quite an improvement, thanks!

@nelsonamaya82 - All that remains then is the somewhat indistinguishable indicators on the Previous mobile nav links, both Freedom and Lively.
